src/types/getKvRecords.d.ts

```diff
@@ -8,11 +8,22 @@ interface GetKvRecordsRequestFunctionParams extends GetKvRecordsRequestParams {
   client: Client;
 }
 
+type AddressTag = {
+  caseSensitive: boolean;
+  id: number;
+  key: string;
+  type: number;
+  val: string;
+};
+
 interface GetKvRecordsData {
   records: {
-    id: string;
-    [key: string]: string;
+    caseSensitive: boolean;
+    id: number;
+    key: string;
+    type: number;
+    val: string;
```

Comment:
Why not use the `AddressTag` type you just defined above?

Reply:
oops [235fd9e](https://github.com/GridPlus/gridplus-sdk/pull/525/commits/235fd9e7f44a6044fb7e5e6f850b64af23c5b4fc)